### PR TITLE
Completed "Win32 C++ app with Visual Composition"

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/samples/webview2samplewincomp?branch=pr-en-us-2006

This PR completes this article, adding a screenshot of the running app.  This article is like the Readme but more detailed steps. 
 Includes steps about installing Windows 10 SDK.

Compare the linked Readme:
https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2SampleWinComp#readme

### CODE REVIEW

File: microsoft-edge/webview2/samples/webview2samplewincomp.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ ms.author: msedgedevrel
...
 # WebView2 sample: Win32 C++ app with Visual Composition
 
 This WebView2 sample demonstrates creating an application that embeds a WebView2 control within a Win32 native application.
 
 It is built as a Win32 Visual Studio 2019 project and makes use of both C++ and HTML/CSS/JavaScript in the WebView2 environment.
 
-It also uses Windows Runtime Composition APIs (also called the Visual layer) to take avantage of the Windows UI features and create a better look, feel, and functionality in C++ Win32 applications.
+It also uses Windows Runtime Composition APIs (also called the Visual layer) to take advantage of the Windows UI features and create a better look, feel, and functionality in C++ Win32 applications.
 
 Directory &amp; .sln: **WebView2SampleWinComp/WebView2SampleWinComp.sln**.
```
Comment (vchapel) [inline microsoft-edge/webview2/samples/webview2samplewincomp.md:19]:
```suggestion
Directory &amp; `.sln`: **WebView2SampleWinComp/WebView2SampleWinComp.sln**.
```

Comment (vchapel) [reply microsoft-edge/webview2/samples/webview2samplewincomp.md:19]: I think file names and extensions should have some sort of formatting ... if this isn't the standard I'm sure you will fix it.

File: microsoft-edge/webview2/samples/webview2samplewincomp.md [hunk 2]
```diff
@@ -73,159 +73,113 @@ Microsoft Visual Studio is required.  Microsoft Visual Studio Code is not suppor
...
 
 <!-- ====================================================================== -->
-## Step 7 - View the opened project
+## Step 6 - Install the Windows SDK
 
-The project opens in Visual Studio:
+By default, this sample app uses the latest Window 10 SDK version you have installed.  There's an issue with Windows 10 SDK, version 2004 (10.0.**19041**.0) that will stop this sample app from building.  If you run into this issue, either install (and retarget this project to use) a later version, such as Windows 10 SDK version 2104 (10.0.**20348**.0), or an earlier version, such as 10.0.**18362**.1.  
 
-![The WebView2SampleWinComp project in Visual Studio.](media/webview2samplewincomp-project-in-sln-explorer.png)
+To install a Windows 10 SDK:
```
Comment (vchapel) [inline microsoft-edge/webview2/samples/webview2samplewincomp.md:92]: What about Windows 11?  Min worked with Win 11 and the 10.0.17763.0 Windows SDK version.

File: microsoft-edge/webview2/samples/webview2samplewincomp.md [hunk 3]
```diff
@@ -73,159 +73,113 @@ Microsoft Visual Studio is required.  Microsoft Visual Studio Code is not suppor
...
 
-<!-- ====================================================================== -->
-## Step 9 - Build the project
+1. Click the **Next** button and then follow the prompts.  You can accept the defaults.  At the end of installing, the Windows SDK Welcome screen for whichever version that you selected appears:
 
-At the top of Visual Studio, set the build target, as follows:
+  ![Welcome to the Windows SDK.](webview2samplewincomp-images/welcome-winsdk.png)
 
-1. In the **Solution Configurations** dropdown list, select **Debug** or **Release**.
+1. Click the **Close** button.
```
Comment (vchapel) [inline microsoft-edge/webview2/samples/webview2samplewincomp.md:112]: Numbered steps in "Step 6" are not consecutive because images need to be indented.

**Review by vchapel — APPROVED** [review]
LGTM!

File: microsoft-edge/webview2/samples/webview2samplewincomp.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/samples/webview2samplewincomp.md:92]: If I understand, you used some version of Visual Studio to build this project targeting Win10 (not Win11), and then ran the resulting Win10 app on Win11.  17763 is a Win10 SDK:
https://developer.microsoft.com/en-us/windows/downloads/sdk-archive/

Comment (mikehoffms) [reply microsoft-edge/webview2/samples/webview2samplewincomp.md:92]: Not sure if updating this article to target win11 is out of scope for this PR.

**Pull request merged**